Extension/src/LanguageServer/client.ts

```diff
@@ -194,6 +194,21 @@ interface WorkspaceFolderParams {
     workspaceFolderUri?: string;
 }
 
+interface SelectionParams {
+    uri: string;
+    range: Range;
+}
+
+export interface VsCodeUriAndRange {
```

Comment:
Minor nit: Could you move this to the file that uses it?  (It looks like only one file?)   Here, it's in the middle of serializable classes, and the fields suggest this isn't serializable.  Also, client.ts is getting pretty bloated.

Reply:
It's only used in client.ts

Reply:
I think it was intended to possibly be used in other places, but I might get rid of it in my subsequent PR since it involves the the rename data which I'm probably going to get rid of so I can get rid of this type then.